context/app/static/js/components/detailPage/provenance/ProvSection/ProvSection.jsx

```diff
@@ -1,12 +1,15 @@
 import React from 'react';
+import InfoIcon from '@material-ui/icons/Info';
+
 import { useAppContext, useFlaskDataContext } from 'js/components/Contexts';
 import useProvData from 'js/hooks/useProvData';
 import { Alert } from 'js/shared-styles/alerts';
-import SectionHeader from 'js/shared-styles/sections/SectionHeader';
+import { SecondaryBackgroundTooltip } from 'js/shared-styles/tooltips';
 import { DetailPageSection } from 'js/components/detailPage/style';
+import { StyledSectionHeader } from './style';
 import ProvTabs from '../ProvTabs';
 
-function ProvSection({ iconTooltipText }) {
+function ProvSection() {
```

Comment:
@NickAkhmetov I agree with you, this is one of the tooltips I had trouble coming up with the language for. The main reason for this tooltip is because some users didn't know the definition of provenance when they were on that section. Do you have any suggestions for improving this tooltip's language?